### PR TITLE
CRM: 3441 - clean up Data Tools page

### PR DESCRIPTION
**Pull request by tbradsha**
Resolves Automattic/zero-bs-crm#3441 - clean up Data Tools page

## Proposed changes:

This PR cleans up the Data Tools page a bit (perfection is out of scope). Screenshots are most illustrative.

Before (free):
image

After (free):
image

Before (white label):
image

After (white label):
image

Before (CSV Importer Pro):
image

After (CSV Importer Pro):
image

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Testing instructions:

Go to this page: `/wp-admin/admin.php?page=zerobscrm-datatools`

Verify things make sense.

### CODE REVIEW

File: projects/plugins/crm/admin/dashboard/main.page.php [hunk 1]
```diff
@@ -72,7 +72,7 @@ function jpcrm_render_dashboard_page() {
 			'count'          => $count,
 			'backfill_count' => $backfill_count,
 			'contact_status' => $contact_status,
-			'link'           => false,
+			'link'           => jpcrm_esc_link( $zbs->slugs['managecontacts'] . '&quickfilters=status_' . $contact_status ),
```
Comment (gogdzl) [inline projects/plugins/crm/admin/dashboard/main.page.php]: I think this got mixed up with another branch.

File: projects/plugins/crm/changelog/fix-crm-3440-add_links_to_dash_sales_funnel [hunk 2]
```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: added
+
+Dashboard: Sales Funnel now links to contact list view.
```
Comment (gogdzl) [inline projects/plugins/crm/changelog/fix-crm-3440-add_links_to_dash_sales_funnel]: I think this got mixed up with another branch.

**Comment by gogdzl** [comment]
I have tried to build the plugin, tried to run gulp, but this margin is not looking like the screenshots provided: 

image

File: projects/plugins/crm/admin/dashboard/main.page.php (hunk 1 above)
Comment (tbradsha) [reply projects/plugins/crm/admin/dashboard/main.page.php]: Indeed; not sure what happened. Repushed with just the relevant commits.

File: projects/plugins/crm/changelog/fix-crm-3440-add_links_to_dash_sales_funnel (hunk 2 above)
Comment (tbradsha) [reply projects/plugins/crm/changelog/fix-crm-3440-add_links_to_dash_sales_funnel]: Indeed; not sure what happened. Repushed with just the relevant commits.

**Comment by tbradsha** [comment]
Agreed; this is a load order issue due to Semantic UI issues and `CONCATENATE_SCRIPTS`. I've logged it in the parent issue here, but I'm going to call it out of scope for this PR.

**Review by gogdzl — APPROVED** [review]
Given that the margin is out of scope, LGTM!

**Pull request merged**